compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/v2/Scrollbar.desktop.kt

```diff
@@ -0,0 +1,417 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.foundation.v2
```

Comment:
I have an idea. We can use `JvmName` to not force users to change their code.

We can keep the old names and package, and still preserve the backward compatibility on binary level:
```
package androidx.compose.foundation

@JvmName("ScrollbarAdapter")
@Deprecated
interface ScrollbarAdapter1

@JvmName("ScrollbarAdapter2")
interface ScrollbarAdapter
```

We lose source compatibility though, but only in few cases, when users use custom implementation of `ScrollbarAdapter`. After they update version of Compose, they have to change interface to ScrollbarAdapter1, or provide `viewportSize` somehow. But it is better than forcing the majority of users who don't use custom adapters to replace deprecated calls.

Reply:
Good idea but unfortunately `JvmName` doesn't currently work for interfaces: https://youtrack.jetbrains.com/issue/KT-31420/Support-JvmName-on-interface-or-provide-other-interface-evolution-mechanism

Reply:
Can we use it for just rememberScrollbarAdapter/VerticalScrollbar, and create only `v2.ScrollbarAdapter`?

Reply:
Done.